.gitignore

```diff
@@ -17,3 +17,12 @@
 /static/lib
 /static/fonts
 /static/experiments
+static/*
+
```

Comment:
Maybe add .DS_Store here too at some point, I've seen it added a lot around the expfactory repos (*shakes angry old man first* "You MAC users, you!")

Is there a reason for the empty lines? I've never seen that :thinking: